python/res/simulator/batch_simulator_context.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, result_keys, ert, fs, mask, itr):
         """
         super(BatchContext, self).__init__(ert, fs, mask, itr)
         self.result_keys = result_keys
-        self.res_config = ert.resConfig( )
+        self.res_config = ert.resConfig()
```

Comment:
Please revert white space fixes. This PR is touching enough files as is.

Reply:
ok ... just didn't like the emptiness inside :)